app/enrichment/enrich-header.js

```diff
@@ -11,7 +11,7 @@ const enrichHeader = async (paymentRequest, scheme) => {
   paymentRequest.schemeId = scheme?.schemeId
   paymentRequest.agreementNumber = paymentRequest.agreementNumber ?? paymentRequest.contractNumber
   paymentRequest.invoiceNumber = createInvoiceNumber(paymentRequest)
-  paymentRequest.ledger = AP
+  paymentRequest.ledger = paymentRequest.ledger ?? AP
```

Comment:
Why have you changed this as well?

Reply:
Seemed a quick win.  We're about to get manual invoices which can be AP or AR, so overriding with AP wouldn't work.